### DISCUSSION TITLE
如何添加.jar格式的整合包？

### DISCUSSION THREAD

**Discussion by trustedinster** [Meloong-Git/PCL#694] [提问] [answered]
我有一个网上找的整合包，.jar格式的，pcl2不能直接导入，又没能在pcl2中找到，我该怎么办？

**Comment by xxtg666** [comment]
整合包还有jar格式？建议直接解压一下看看

**Comment by WForst-Breeze** [comment, marked as answer]
jar格式的真没见过（）可以试试楼上说的直接解压试试，也可以您把整合包文件传上来看看（如果可以的话
